server/src/test/java/org/apache/druid/server/coordinator/ClientCompactionRunnerInfoTest.java

```diff
@@ -21,6 +21,7 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import org.apache.druid.client.indexing.ClientCompactionRunnerInfo;
```

Comment:
Shouldn't need to import here. Test should be in the same package as the target class being tested.